test/IntegrationTests/appsettings.json

```diff
@@ -26,11 +26,13 @@
   "GeneralSettings": {
     "OpenIdWellKnownEndpoint": "http://localhost:5040/authentication/api/v1/openid/",
     "BridgeApiEndpoint": "https://at22.altinn.cloud/sblbridge/authorization/api/",
+    "ApiStorageEndpoint": "http://localhost:5010/storage/api/v1/",
```

Comment:
Environment variable for the new endpoint must also be added to values.yaml in altinn-studio-ops repo

Reply:
Done in PR664